### DISCUSSION TITLE
Will all JavaScript be moved to VelaCP

### DISCUSSION THREAD

**Discussion by kminder** [kubevela/kubevela#1161] [General] [answered]
We are trying to identify the minimal set of dependencies required to use KubeVela.  Our understanding is that the console UI will be moving from the KubeVela project to the VelaCP project.  Can we assume that the KubeVela repo will contain no JavaScript when this change is complete?  Can you also share the anticipated timeline for this change?

**Comment by wonderflow** [comment, marked as answer]
>  Can we assume that the KubeVela repo will contain no JavaScript when this change is complete? 

Yes.

> Can you also share the anticipated timeline for this change?

We're planing to move all `references` folder to `velacp` before this June. （modified）
